playkit/src/main/java/com/kaltura/playkit/player/SubtitleStyleSettings.java

```diff
@@ -0,0 +1,182 @@
+        return this;
+    }
+
+    public SubtitleStyleSettings setSubtitleEdgeType(SubtitleStyleEdgeType subtitleEdgeType) {
+        switch (subtitleEdgeType) {
+            case EDGE_TYPE_NONE:
+                this.subtitleEdgeType = CaptionStyleCompat.EDGE_TYPE_NONE;
+                break;
+            case EDGE_TYPE_OUTLINE:
+                this.subtitleEdgeType = CaptionStyleCompat.EDGE_TYPE_OUTLINE;
+                break;
+            case EDGE_TYPE_DROP_SHADOW:
+                this.subtitleEdgeType = CaptionStyleCompat.EDGE_TYPE_DROP_SHADOW;
+                break;
+            case EDGE_TYPE_RAISED:
+                this.subtitleEdgeType = CaptionStyleCompat.EDGE_TYPE_RAISED;
+                break;
+            case EDGE_TYPE_DEPRESSED:
+                this.subtitleEdgeType = CaptionStyleCompat.EDGE_TYPE_DEPRESSED;
+                break;
+            default:
+                this.subtitleEdgeType = CaptionStyleCompat.EDGE_TYPE_NONE;
+                break;
+        }
+        return this;
+    }
+
+    public SubtitleStyleSettings setSubtitleTextSizeFraction(SubtitleTextSizeFraction subtitleTextSizeFraction) {
+        switch (subtitleTextSizeFraction) {
+            case SUBTITLE_FRACTION_50:
+                this.subtitleTextSizeFraction = fraction50;
+                break;
+            case SUBTITLE_FRACTION_75:
+                this.subtitleTextSizeFraction = fraction75;
+                break;
+            case SUBTITLE_FRACTION_100:
+                this.subtitleTextSizeFraction = fraction100;
+                break;
+            case SUBTITLE_FRACTION_125:
+                this.subtitleTextSizeFraction = fraction125;
+                break;
+            case SUBTITLE_FRACTION_150:
+                this.subtitleTextSizeFraction = fraction150;
+                break;
+            case SUBTITLE_FRACTION_200:
+                this.subtitleTextSizeFraction = fraction200;
+                break;
+            default:
+                this.subtitleTextSizeFraction = fraction100;
+                break;
+        }
+        return this;
+    }
+
+    public SubtitleStyleSettings setSubtitleTypeface(SubtitleStyleTypeface subtitleStyleTypeface) {
+        switch (subtitleStyleTypeface) {
+            case DEFAULT:
+                subtitleTypeface = Typeface.DEFAULT;
+                break;
+            case DEFAULT_BOLD:
+                subtitleTypeface = Typeface.DEFAULT_BOLD;
+                break;
+            case MONOSPACE:
+                subtitleTypeface = Typeface.MONOSPACE;
+                break;
+            case SERIF:
+                subtitleTypeface = Typeface.SERIF;
+                break;
+            case SANS_SERIF:
+                subtitleTypeface = Typeface.SANS_SERIF;
+                break;
+            default:
+                subtitleTypeface = Typeface.DEFAULT;
+                break;
+        }
+        return this;
+    }
+
+    protected CaptionStyleCompat toCaptionStyles() {
```

Comment:
Should be `toCaptionStyle()` (singular).
And it should be package-private, not protected.

Reply:
Fixed this issue. Access is now public because private function will not be accessible in ExoPlayerWrapper.